docs/api.rst

```diff
@@ -0,0 +1,28 @@
+API reference
+=============
+
+Schema
+------
+.. autoclass:: django_graph_api.Schema
```

Comment:
I am not a huge fan of docstrings as documentation personally, since the things you want to say about a class for a wider audience may be larger / different than what is neat / useful to put into a class or method docstring.

That being said, the docstrings you've written are all a great starting place either way, and if we wanted to expand on them later it would be easy to swap something else in here.